src/map.rs

```diff
@@ -257,6 +267,84 @@ impl Map {
     }
 }
 
+/// Represents the way tiles are laid out in a map.
```

Comment:
Documentation copied from `Orientation` enum. For `StaggerIndex`, I would suggest:

```
// Specifies whether the odd or even rows/columns are shifted half a tile
// right/down. Only applies to Staggered and Hexagonal map orientations.
```